Design Patterns/Design Patterns/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AbstractFactory.Classes.Factories;
+using AbstractFactory.Interfaces;
+
+namespace AbstractFactory
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Factory can be changed without any negative consequences to code
+            // This results in different behaviour with almost the same code
+            IAbstractCarFactory factory = new AbstractExtraFactory();
```

Comment:
Can you please add a method that will dynamically initialize factory using enum like CarConfiguration to show that we define behaviour of a factory in a runtime.

Reply:
Fixed